src/persistence/existingNamespaceUpdateInterceptor.js

```diff
@@ -6,11 +6,36 @@ export default function existingNamespaceUpdateInterceptor(openmct, usersNamespa
             return shouldCheck;
         },
         invoke: (identifier, object) => {
+            // check for old user namespace style
             if (object.location === usersNamespace.key) {
                 object.location = usersNamespace.id;
                 openmct.objects.mutate(object, 'location', usersNamespace.id);
             }
 
+            // check for old composition style
+            if (object.composition?.length) {
+                let updatedComposition = false;
+
+                object.composition = object.composition.map((keystring) => {
+                    if (typeof keystring === 'string') {
+                        updatedComposition = true;
```

Comment:
Side effect in a map!?! You savage.

I'll allow it.

Reply:
ha, I didn't even think about that... next time.